samples/net/wifi/README.rst

```diff
@@ -0,0 +1,50 @@
+.. _wifi:
+
+Wi-Fi sample
+############
+
+Overview
+********
+
+This sample enables testing Wi-Fi drivers for various boards.
```

Comment:
This sample allows testing Wi-Fi drivers for various boards by
    enabling the Wi-Fi shell module that provides a set of commands:
    scan, connect, and disconnect.  It also enables the net_shell module
    to verify net_if settings.